docs/reference/settings/ml-settings.asciidoc

```diff
@@ -80,6 +80,11 @@ total memory of the machine, not current free memory. Jobs are not allocated to
 a node if doing so would cause the estimated memory use of {ml} jobs to exceed
 the limit.
 
+`xpack.ml.max_ml_node_size`::
+(<<static-cluster-setting,Static>>)
```

Comment:
It’s dynamic now (changed earlier today)

Reply:
Yup, I have another PR pending (https://github.com/elastic/elasticsearch/pull/66286), which will go in after this one.